### PR TITLE
Update docs - part 3 (New ToggleGroup)

### PR DESCRIPTION
**Pull request by benoitgrelard**
----------

- [x] Change base branch to `main` once #120 is merged. 

----------

This PR also update Accordion docs.

### CODE REVIEW

File: components/PrimitivesNav.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export function PrimitivesNav() {
         <NavHeading id="site-nav-components">Components</NavHeading>
         <NavList>
           {componentsPages.map((page) => {
-            const idWithoutVersion = removeVersionFromId(page.version, page.id);
+            const idWithoutVersion = removeVersionFromId(page.version, page.id) + '/';
```
Comment (benoitgrelard) [inline components/PrimitivesNav.tsx:35]: Had to do this to make sure we don't highlight `toggle` in the nav when `toggle-group` is the active page.

File: layouts/DocsLayout.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default function DocsLayout({ children, frontMatter }: LayoutProps) {
 
   const currentPageId = router.pathname.substr(1);
   const currentPageIndex = productPages.findIndex((page) =>
-    currentPageId.includes(removeVersionFromId(page.version, page.id))
+    currentPageId.includes(removeVersionFromId(page.version, page.id) + '/')
```
Comment (benoitgrelard) [inline layouts/DocsLayout.tsx:54]: Same thing here, the pagination would pick the wrong previous/next when on `toggle-group` page.

**Review by peduarte — APPROVED** [review]

File: pages/primitives/docs/components/toggle-group/0.0.5/index.mdx [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+    },
+    {
+      name: 'onValueChange',
+      required: false,
+      type: '(value: string[]) => void',
+      typeSimple: 'function',
+      description: (
+        <span>
+          Event handler called when the pressed state of an item
+          changes (when <Code>type</Code> is <Code>"multiple"</Code>).
```
Comment (jjenzz) [inline pages/primitives/docs/components/toggle-group/0.0.5/index.mdx:182]:
```suggestion
          Event handler called when the pressed state of an item
          changes and <Code>type</Code> is <Code>"multiple"</Code>.
```

**Review by jjenzz — APPROVED** [review]
Small suggestion below but feel free to ignore if you don't agree

**Comment by benoitgrelard** [comment]
> Small suggestion below but feel free to ignore if you don't agree. I just thought there were too many "when"s for me when reading haha.

YES! I'll update the next PR too then.

**Pull request merged**